From 71c0f603185ab5073279f800e8a4f045c7542751 Mon Sep 17 00:00:00 2001
From: Noam Lessner <nlessner@habana.ai>
Date: Mon, 03 Feb 2020 16:30:23 +0200
Subject: [PATCH] [SW-8229] goya2: add uint64 constant

Explicit define the 64bit constant so the compiler will
make sure its used as a 64bit constant

Change-Id: I56d49037af468658a50b1c8f93010df839ec7def
Signed-off-by: Noam Lessner <nlessner@habana.ai>
---

diff --git a/include/arcboot.h b/include/arcboot.h
index 03c9df6..4f57cb3 100644
--- a/include/arcboot.h
+++ b/include/arcboot.h
@@ -21,7 +21,7 @@
 
 #define SPI_FLASH_BLOCK_SIZE			1024
 
-#define HL_SRAM_BASE				0x7FF0000000
+#define HL_SRAM_BASE				0x7FF0000000ULL
 #define HL_SDRAM_BASE				0x10000000
 #define HL_FLASH_OFFSET				0xA000000
 #define HL_ICCM_OFFSET				0xB020000
